### PR TITLE
First cut of Interoperability.adoc

### PR DESCRIPTION
**Pull request by smillidge**
This needs a review. Still includes references to JAX-RPC as I don't know the new name for that as it should really be pruned.

Signed-off-by: smillidge <steve.millidge@payara.fish>

### CODE REVIEW

File: specification/src/main/asciidoc/platform/Interoperability.adoc [hunk 1]
```diff
@@ -77,27 +77,27 @@ transport protocols for the Internet. TCP/IP is supported by Java SE and
...
-Java EE web container must be capable of advertising its HTTPS service
+Jakarta EE web container must be capable of advertising its HTTPS service
 on the standard HTTPS port, port 443. Note that SSL 3.0 and only TLS 1.0
-are required as part of the EJB interoperability protocol in the EJB
+are required as part of the Jakarta Enterprise Beans interoperability protocol in the Jakarta Enterprise Beans
 specification.
 * SOAP 1.1—SOAP is a presentation layer
 protocol for the exchange of XML messages. Support for SOAP layered on
-HTTP is required, as described in the JAX-RPC and JAX-WS specifications.
+HTTP is required, as described in the JAX-RPC and Jakarta XML Web Services specifications.
```
Comment (ivargrimstad) [inline specification/src/main/asciidoc/platform/Interoperability.adoc]: Jakarta XML-based RPC

File: specification/src/main/asciidoc/platform/Interoperability.adoc [hunk 2]
```diff
@@ -77,27 +77,27 @@ transport protocols for the Internet. TCP/IP is supported by Java SE and
...
 * SOAP 1.2—SOAP 1.2 is the version of the SOAP
-protocol standardized through W3C and supported by JAX-WS.
+protocol standardized through W3C and supported by Jakarta XML Web Services.
 * WS-I Basic Profile 1.1—The WS-I Basic
 Profile, in combination with the Simple SOAP Binding Profile and
 Attachment Profile, describes interoperability requirements for the use
 of SOAP 1.1, WSDL 1.1, and MIME-based SOAP with Attachments. It is
-required by the JAX-RPC and JAX-WS specifications.
```
Comment (ivargrimstad) [inline specification/src/main/asciidoc/platform/Interoperability.adoc]: Jakarta XML-based RPC

File: specification/src/main/asciidoc/platform/Interoperability.adoc [hunk 3]
```diff
@@ -1,55 +1,55 @@
 == [[a2845]]Interoperability
 
 This chapter describes the interoperability
-requirements for the Java™ Platform, Enterprise Edition (Java EE).
+requirements for the Jakarta EE™ Platform.
```
[the remark this answers is not in the document]
Comment (smillidge) [reply specification/src/main/asciidoc/platform/Interoperability.adoc]: Done

File: specification/src/main/asciidoc/platform/Interoperability.adoc (hunk 2 above)
Comment (smillidge) [reply specification/src/main/asciidoc/platform/Interoperability.adoc]: Done

File: specification/src/main/asciidoc/platform/Interoperability.adoc (hunk 1 above)
Comment (smillidge) [reply specification/src/main/asciidoc/platform/Interoperability.adoc]: Done

File: specification/src/main/asciidoc/platform/Interoperability.adoc [hunk 4]
```diff
@@ -77,27 +77,27 @@ transport protocols for the Internet. TCP/IP is supported by Java SE and
...
-Java EE web container must be capable of advertising its HTTPS service
+Jakarta EE web container must be capable of advertising its HTTPS service
 on the standard HTTPS port, port 443. Note that SSL 3.0 and only TLS 1.0
-are required as part of the EJB interoperability protocol in the EJB
+are required as part of the Jakarta Enterprise Beans interoperability protocol in the Jakarta Enterprise Beans
 specification.
 * SOAP 1.1—SOAP is a presentation layer
 protocol for the exchange of XML messages. Support for SOAP layered on
-HTTP is required, as described in the JAX-RPC and JAX-WS specifications.
+HTTP is required, as described in the Jakarta XML-based RPC and Jakarta XML Web Services specifications.
```
Comment (bshannon) [inline specification/src/main/asciidoc/platform/Interoperability.adoc]: Jakarta XML Web Services is not part of Jakarta EE 8.
This should continue to refer to JAX-WS.

**Pull request merged**